pkg/universe.dagger.io/alpha/uffizzi/uffizzi.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+UFFIZZI_PASSWORD="$(cat /run/secrets/uffizzi_password)"
+export UFFIZZI_PASSWORD
```

Comment:
Why not directly export the value in one line?

Reply:
Good idea